environments/icds-cas/known_hosts

```diff
@@ -25,19 +25,25 @@
 100.71.184.12, ecdsa-sha2-nistp256 AAAAE2VjZHNhLXNoYTItbmlzdHAyNTYAAAAIbmlzdHAyNTYAAABBBCokfTDeDQoFx4hGwPMVzVacGvD/3zmcD99zdG6z09xn0hVES+iJLXu/6EnaOwTbw1odRGH1m7J7sjribOCGixw=
 100.71.184.12, ssh-rsa AAAAB3NzaC1yc2EAAAADAQABAAABAQDNLwa7WReBrjf+wgGgZj3+5UNGXfDuPIhbj4DHihBmXlq4bL1fWDL41J2gbccU7+9n888NL6OiaowsIDlSrhuvt0Twxr3UxKRNkMp9GERLWK0feOTkfE6OUe+YvCuiLjOHLLOgcw3o2NIsCIz7S8aumKnW+UHRF5s6apHOjNJepCu4gVVaRqRm6lRB6YrlMXLvLcgdDAEHmn3An6fVYL8GH1wtwoZCXg9z3ITSFXu5733e9kstatzBdkKKP4kD6rP0gYRaQ7Q/WUuF3ezOL21sQRbRzouVKhd1FB2Jt+NfXaD8NrGiLvWqN3aB06xUjwGGZvdpLCVtfVoS9hwKXGgN
+100.71.184.120, ecdsa-sha2-nistp256 AAAAE2VjZHNhLXNoYTItbmlzdHAyNTYAAAAIbmlzdHAyNTYAAABBBM0oeWvi1nCIa6Y2c2sgM6EesGLOekWJcZ685Xk3jPkg9tr1GAIvFd12RGL1kqJXtFFbGEiuyt6oT9vw3P7LtB8=
+100.71.184.120, ssh-rsa AAAAB3NzaC1yc2EAAAADAQABAAABAQDaDEeZORODV3NBRiZ+u4OLWPPi65S7+ngmkr5tWh32QHjnJiKeBtZ95FMqeO/EdHFKUqcpdGFfSrqXodYw6hXh8x+Bcs58miovywHZ4yvrVYC6VxDnqQ/NbBx2hXoeausjaHTPJkarQobmhgK0c02qbLr/bUERcVhsHk4liF2n6jryVSsNRAtMpZe5AkSsFFe+/zTmaGIMvyiUO7McHGjzsp34pnjwsXIGaFfVDYG8YdWzcnatgmHstv7A0ZeSh5Xk+90x+nSw369ISw+yhsbWFacZfmBiGOFUFPcYHVYstzr94r4Iw5KjmgfZaChg2Uw7QpfNSXabKRb1aUmqmuX7
+100.71.184.121, ecdsa-sha2-nistp256 AAAAE2VjZHNhLXNoYTItbmlzdHAyNTYAAAAIbmlzdHAyNTYAAABBBM0oeWvi1nCIa6Y2c2sgM6EesGLOekWJcZ685Xk3jPkg9tr1GAIvFd12RGL1kqJXtFFbGEiuyt6oT9vw3P7LtB8=
+100.71.184.121, ssh-rsa AAAAB3NzaC1yc2EAAAADAQABAAABAQDaDEeZORODV3NBRiZ+u4OLWPPi65S7+ngmkr5tWh32QHjnJiKeBtZ95FMqeO/EdHFKUqcpdGFfSrqXodYw6hXh8x+Bcs58miovywHZ4yvrVYC6VxDnqQ/NbBx2hXoeausjaHTPJkarQobmhgK0c02qbLr/bUERcVhsHk4liF2n6jryVSsNRAtMpZe5AkSsFFe+/zTmaGIMvyiUO7McHGjzsp34pnjwsXIGaFfVDYG8YdWzcnatgmHstv7A0ZeSh5Xk+90x+nSw369ISw+yhsbWFacZfmBiGOFUFPcYHVYstzr94r4Iw5KjmgfZaChg2Uw7QpfNSXabKRb1aUmqmuX7
 100.71.184.122, ecdsa-sha2-nistp256 AAAAE2VjZHNhLXNoYTItbmlzdHAyNTYAAAAIbmlzdHAyNTYAAABBBN9QsurRBHAvh38TRIPq/Nqb13vcZQGIEXdicfMEpW00BghKt5L86mFUPNnoBe9quRpmPGZPQ0TfvG6Mmq0XtYc=
 100.71.184.122, ssh-ed25519 AAAAC3NzaC1lZDI1NTE5AAAAIM59p+54uK1hgWhRzCpzkK0JUVpa8ShhffRnA2ZJWmca
 100.71.184.122, ssh-rsa AAAAB3NzaC1yc2EAAAADAQABAAABAQCx64lpQJajb2FPh38jHiwSmqm836zn8qciwM5+jd7vMke+PTmhANLIR6/Rc2VrIFtsgJNBIUvmmQMjY4mJ+7i2Y2DU44FMclURAd830GP54FC/jw05KLYSoh/cTwvwYLZoGBJm/FeHjouS/5Y2Atm6QqN+OjnA+eXtcTBSCuoc2C4wxPOxde9wuNzE9sFFuEzRIDNZX6iio21BPmL0y3I2NaKZZ9jKueLKaEF0pBT1fiFnKAtHUl6tMe9F36XnVxy2aidljuIYj9jwv8vLN3kZqTX/uV7otxjGSoo64Ly7uYXUUMWtmR2QwkGkHl0KgOq02ORw7CMr+mUDDYQ+Ieap
-100.71.184.123, ssh-ed25519 AAAAC3NzaC1lZDI1NTE5AAAAIEVnmUrtmcKhtOuaZLQ39TCU7G11frCAyVtKUfDFW1VR
+100.71.184.123, ecdsa-sha2-nistp256 AAAAE2VjZHNhLXNoYTItbmlzdHAyNTYAAAAIbmlzdHAyNTYAAABBBCvcpuEcsJDFcYhFBzazUoHA4IMFjLU+dCWv+IJydGObKgcXOxasAkNQbhDJrGqWwD/pR7/xD+JcmK4L5QSBbkY=
```

Comment:
it appears whatever ssh configuration I have removed `ed25519` keys and replaced them with these `ecdsa` keys, thought not all of them. I'm fairly confused why it did that